hack/common.sh

```diff
@@ -658,6 +658,13 @@ EOF
 }
 readonly -f os::build::save_version_vars
 
+# os::build::get_product_vars exports variables that we expect to change
+# depending on the distribution of Origin
+function os::build::get_product_vars() {
+  export OS_IMAGE_PREFIX="${OS_IMAGE_PREFIX:-"openshift/origin"}"
+  export OS_DEFAULT_IMAGE_STREAMS="${OS_DEFAULT_IMAGE_STREAMS:-"centos7"}"
```

Comment:
The second one. Sure. I was hoping to use the first one in `hack/build-images` later so we could build fake OCP images without OSBS for tests for OSE repo